### ISSUE TITLE
Failing to get APNonce on iPhone 12 (iOS 15 Beta 3)

### ISSUE THREAD

**Issue by K4CZP3R** [airsquared/blobsaver#312]
**Describe the bug**
When trying to read APNonce on iOS 15 Beta 3 (iphone 12) it fails even before getting into the recovery mode. Reading APNonce in normal mode seems to be failing.

**Desktop:**
 - OS Version: Windows 11 Insider Preview and MacOS monterey beta 3
 - iOS Device: iPhone 12 with iOS 15 beta 3

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked the blobsaver wiki to see if there is already a solution to my problem
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
java.lang.NullPointerException: Cannot invoke "com.sun.jna.Pointer.getByteArray(long, int)" because the return value of "airsquared.blobsaver.app.natives.Libplist.getDataPtr(com.sun.jna.Pointer, com.sun.jna.ptr.IntByReference)" is null
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.plistDataToString(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.getApNonceNormalMode(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

```

**Comment by hubert3** [comment]
Same on iPhone 12 Pro Max iOS 15 Beta 4

**Comment by K4CZP3R** [comment]
I've got solution from CoocooFroggy on Reddit:

---
The way blobsaver fetches AP Nonce and BootNonce in normal mode on the 15 betas is broken right now, you should do it manually with
`ideviceinfo from libimobiledevice`
If you have MacOS you can do:
1. `brew install libimobiledevice`
2. Then `ideviceinfo -k ApNonce` and copy that and decode it from b64, that's your AP Nonce 
4. Reboot the device 
5. `ideviceinfo -k BootNonce` and decode from b64, that's your generator

After decoding it from base64 (using python, add `0x` for the generator)

**Comment by airsquared** [comment]
This has been fixed in v3.0.2, you should now be able to get the information with blobsaver automatically again.
